shared/components/SwapProgress/SwapProgress.js

```diff
@@ -61,7 +61,7 @@ export default class SwapProgress extends Component {
         )
       case 5:
         return (
-          <FormattedMessage id="SwapProgress56" defaultMessage="Creating Ethereum Contract. \n Please wait, it can take a few minutes" />
+          <FormattedMessage id="SwapProgress56" defaultMessage="Creating Ethereum Contract. {br} Please wait, it can take a few minutes" values={{ br: <br /> }} />
```

Comment:
Зачем пробелы, когда тут перенос строки.
```suggestion
          <FormattedMessage id="SwapProgress56" defaultMessage="Creating Ethereum Contract.{br}Please wait, it can take a few minutes" values={{ br: <br /> }} />
```